contrib/gocql/gocql/gocql_test.go

```diff
@@ -107,11 +109,13 @@ func TestChildWrapperSpan(t *testing.T) {
 		pSpan = spans[0]
 	}
 	assert.Equal(pSpan.OperationName(), "parentSpan")
-	assert.Equal(childSpan.ParentID(), pSpan.SpanID)
+	assert.Equal(childSpan.ParentID(), pSpan.SpanID())
 	assert.Equal(childSpan.OperationName(), ext.CassandraQuery)
 	assert.Equal(childSpan.Tag(ext.ResourceName), "SELECT * from trace.person")
 	assert.Equal(childSpan.Tag(ext.CassandraKeyspace), "trace")
-	assert.Equal(childSpan.Tag(ext.TargetPort), "9042")
-	assert.Equal(childSpan.Tag(ext.TargetHost), "127.0.0.1")
-	assert.Equal(childSpan.Tag(ext.CassandraCluster), "datacenter1")
+	if iter.Host() != nil {
```

Comment:
Why would `iter.Host()` be nil here ? Should we call `assert.NotNil()` ?

Reply:
I honestly don't know the condition under which this value is `nil` or not. I have assumed it could be environment specific so I have thought that maybe this should be the approach.

If you know better than me I am happy to change.

Reply:
P.S. It actually _is_ `nil` here.

Reply:
It is not however `nil` in the other test (`TestErrorWrapper`) on my local machine. Sorry for the triple-comment.

Reply:
Hey, this is due to a gocql bug. `iter.Host()` used to always return nil, until this fix was merged to gocql library: https://github.com/gocql/gocql/pull/918
The if was to prevent a break at first for the next gocql release

Reply:
Thanks @furmmon. I have just discovered this myself. I have debugged this and found that in this specific case, gocql exits because of no retry policy and doesn't set the host. I have [opened a PR](https://github.com/gocql/gocql/pull/1126) on the repository with a fix just now.